install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -269,3 +269,21 @@ subjects:
   - kind: ServiceAccount
     name: machine-api-operator
     namespace: openshift-machine-api
+
+---
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-k8s
+    namespace: openshift-monitoring
```

Comment:
as mentioned in https://github.com/openshift/machine-api-operator/pull/340/files/e25d72e68810159a0fcfa547ad02c37cc1130981#r304315286, this should be your namespace, i.e. `openshift-machine-api`.

Reply:
ack